components/rss/app/rss.app.ts

```diff
@@ -93,6 +93,16 @@ export default defineApp({
         feedparser.on("end", resolve);
         feedparser.on("readable", function (this: FeedParser) {
           let item: any = this.read();
+          /* 
+          Valid escaped entries in RSS are being stripped out, see issue: https://github.com/danmactough/node-feedparser/issues/243
+          Author suggests using the values below, so I check for them, if they exist, overwrite title.
+          */
+          if(item['atom:title'] && item['atom:title']['#']) {
+            item.title = item['atom:title']['#'];
+          } else if(item['rss:title'] && item['rss:title']['#']) {
+            item.title = item['rss:title']['#'];
+          }
```

Comment:
Hi, @cfjedimaster. I haven't tested, but it looks like you need to check that `item` exists here.
```suggestion
          if(item && item['atom:title'] && item['atom:title']['#']) {
            item.title = item['atom:title']['#'];
          } else if(item && item['rss:title'] && item['rss:title']['#']) {
            item.title = item['rss:title']['#'];
          }
```

Reply:
I'm probably a few days from being able to do this, but your mod looks good and if you were to edit my PR, I wouldn't complain (not even sure if that's a thing).